controllers/UserManagement/OrganisationCreateController.js

```diff
@@ -0,0 +1,52 @@
+const BaseController = require('../BaseController');
+const {postRequest} = require('../../services/UserManagement/UserActionsService');
+const teamPermissionsObjectBuilder = require('../../helpers/teamPermissionsObjectBuilder');
+
+class OrganisationCreateController extends BaseController {
+
+  async saveValues(req, res, next) {
+
+    // Get the values from the form
+    const organisationName = req.form.values['organisationName'];
+
+    try {
+      req.sessionModel.set('addOrgAttempt', true);
+
+      const orgResult = await postRequest({
+        ...this.getOptions(req),
+        url: '/admin/organisations',
+      }, { organisationName: organisationName });
+
+      const createdOrg = orgResult.organisation;
+
+      const defaultTeamName = 'Administrators';
+      const defaultTeamPermissions = ['birth', 'death', 'marriage', 'partnership', 'user-management' ];
```

Comment:
In the case of the default Administrators team, they should have just "user-management". Sometimes they will have search permissions, but this is atypical. At the moment, by default they will be able to search everything which may not be the agreement signed with the organisation.